app/services/xccdf_tailoring_file.rb

```diff
@@ -20,13 +20,31 @@ def builder
     @builder ||= create_builder do |xml|
       xml[XCCDF].benchmark(id: @profile.benchmark.ref_id)
```

Comment:
It looks like the benchmark elements needs a `href` to be validated by the oscap parser, otherwise you get the following error when testing:

```
OpenSCAP Error: File 'tailoring_c2s_with_backend.xml' line 3: Element '{http://checklists.nist.gov/xccdf/1.2}benchmark': The attribute 'href' is required but missing.
 [xccdf_session.c:614]
Invalid XCCDF Tailoring (1.2) content in tailoring_c2s_with_backend.xml. [oscap_source.c:343]
Invalid XCCDF Tailoring (1.2) content in tailoring_c2s_with_backend.xml [xccdf_session.c:1172]
```

Reply:
I've added href now - it's not actually used for anything, but I'm using the inferred_os_major_version on a benchmark to create it in the format of `ssg-rhel7-ds.xml`.